### PR TITLE
Bugzapper 

### PR DESCRIPTION
**Pull request by Aurorablade**
possiblely Fixes https://github.com/ParadiseSS13/Paradise/issues/5705

I am trying to fix the xeno surgery bug, its a runtime on organ_internal line 56, for some reason it dosen't think the xenomorph owns the xeno organ..and i am not sure WHY...

may add other fixes to this..unsure yet..

:cl: Fethas
bugfix: Fixes human jetpacks..maybe...
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -68,6 +68,8 @@
 	if(..())
 		return 1
 
+	if(pulling && istype(pulling, /mob/living/carbon/human))
+		stop_pulling()
```
Comment (Krausus) [inline code/modules/mob/living/carbon/human/human_movement.dm]: Also, it doesn't actually fix it - mutual pulling will make the `return 1` occur before this even executes.

Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human_movement.dm]: also! `ishuman(pulling)` would be the better/faster typecheck.

;w;

Comment (Aurorablade) [inline code/modules/mob/living/carbon/human/human_movement.dm]: oh no its why i post it here. Anyone have a better solution for this or whats going on with xeno organs shit?

Comment (Markolie) [inline code/modules/mob/living/carbon/human/human_movement.dm]: https://github.com/ParadiseSS13/Paradise/pull/5801 fixes the Xeno organ runtime.

**Comment by Aurorablade** [comment]
i am probaly done with this...

**Comment by Aurorablade** [comment]
yeah ready fior review.

**Pull request merged**